generator/tests/cli_test.py

```diff
@@ -0,0 +1,16 @@
+import pytest
```

Comment:
nice! does the test run without specifying the input vars in the function's signature? I haven't tried it so I am curious ☺️

Reply:
no it does not,... fixed in https://github.com/InPreD/samplesheet_generator/pull/1/commits/5290dbf606f17524e1f6b454a0179eb62c1ba0da